crates/common/download/Cargo.toml

```diff
@@ -23,7 +23,7 @@ url = "2.2"
 [dev-dependencies]
 anyhow = "1.0"
 mockito = "0.31"
-regex = "1.5"
+regex = "1.5.5"
```

Comment:
From what I see, `1.5.5` is the latest version of the `regex` crate. And my understanding was that when a version like `1.5` without the third version number is specified in `Cargo.toml`, the latest version of that crate would be picked up(`1.5.5` in this case). So, why didn't that work?